### PR TITLE
Append use cookie option

### PR DESCRIPTION
**Pull request by k2tzumi**
refs #548

### CODE REVIEW

File: http.go [hunk 1]
```diff
@@ -308,6 +349,12 @@ func (rnr *httpRunner) Run(ctx context.Context, r *httpRequest) error {
 			return err
 		}
 		r.setContentTypeHeader(req)
+
+		// Override useCookie
+		if r.useCookie == nil && rnr.useCookie != nil && *rnr.useCookie {
+			r.useCookie = rnr.useCookie
+		}
```
Comment (k2tzumi) [inline http.go:356]: httpRunner setting is enabled only if the useCookie option is unset in the req section

File: http.go [hunk 2]
```diff
@@ -55,6 +56,7 @@ type httpRunner struct {
 	cert              []byte
 	key               []byte
 	skipVerify        bool
+	useCookie         *bool
```
Comment (k2tzumi) [inline http.go:59]: The setting of httpRunner may not need to be nil, but httpRequest handles nil, so we have matched it.

File: testdata/book/cookie_in_requests_automatically.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+desc: Cookie in requests automatically
+runners:
+  req:
+    endpoint: ${HTTPBIN_END_POINT:-https://httpbin.org/}
+    useCookie: true
```
Comment (k1LoW) [inline testdata/book/cookie_in_requests_automatically.yml:5]: 👀

**Review by k1LoW — APPROVED** [review]
LGTM!

**Pull request merged**